.github/workflows/windows.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
           path: |
             ${{ env.GOCACHE }}
             ${{ env.GOMODCACHE }}
-          key: ${{ runner.os }}-buf-windows-${{ hashFiles('**/go.sum', '.github/workflows/windows.yaml') }}
+          key: ${{ runner.os }}-buf-windows-${{ hashFiles('**/go.sum', 'windows/**') }}
```

Comment:
Our other GitHub Actions workflow files have a leading `/`, so we should be consistent.